_pages/get-involved.md

```diff
@@ -9,6 +9,7 @@ permalink: /get-involved/
 The [Democratic Socialists of America](https://www.dsausa.org/) is the largest socialist organization in the United States because we’re a member-driven mass organization. We are a political and activist organization, not a party. We believe that working people should run both the economy and civil society, and we show our commitment to this principle by being an organization of, by, and for the working class.
 
 [Join DSA](https://act.dsausa.org/donate/membership/?source=Baton%20Rouge) today!
+[Click here](https://actionnetwork.org/forms/join-brdsa) to join Baton Rouge DSA's email list.
```

Comment:
I think removing the "click here" language would be best. "Click" assumes the user is using a mouse, which isn't the case for mobile devices or accessibility-focused tools, e.g. screen readers. I'd recommend something more like:

```suggestion
Want to stay in the loop? [Join Baton Rouge DSA’s email list](https://actionnetwork.org/forms/join-brdsa).
```